src/App.js

```diff
@@ -1,29 +1,22 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+// class App extends React.Component {
+//   constructor(props) {
+//     super(props);
+//   }
+
+//   render() {
+//     return (
+//       <h1> Calculator</h1>
+//     );
+//   }
+// }
+
+const App = () => (
+  <div>
+    <h1> Calculator</h1>
+    <p>Build it now</p>
+  </div>
+);
```

Comment:
- [ ] Please convert it to a class-based component as the project requirements dictate 🙏 😄 It's so important at this step to practice the class-based react components and experience how they work 😉 
- [ ] Also, please don't leave any commented-out code in order to keep your code clean and your app optimized 👍